app/pages/resource/info/ResourceInfo.js

```diff
@@ -1,13 +1,34 @@
 import React, { PropTypes } from 'react';
+import Col from 'react-bootstrap/lib/Col';
+import Grid from 'react-bootstrap/lib/Grid';
+import Row from 'react-bootstrap/lib/Row';
 
 function ResourceInfo({ resource, unit }) {
   return (
     <div className="resource-info">
-      <h1>
-        <span className="unit-name">{unit.name.fi}</span>
-        {' '}
-        <span className="resource-name">{resource.name.fi}</span>
-      </h1>
+      <header>
+        <h2 className="unit-name">{unit.name.fi}</h2>
+        <h1 className="resource-name">{resource.name.fi}</h1>
+        <h4 className="unit-address">{unit.streetAddress.fi}</h4>
+      </header>
+      <section className="resource-details">
+        <Grid>
+          <Row>
+            <Col xs={12} sm={6}>
+              <figure />
+            </Col>
+            <Col xs={12} sm={6}>
+              <aside>
+                <h3 className="resource-type">{resource.type.name.fi}</h3>
```

Comment:
- [ ] Some resources don't have type (for example "Vihreiden ryhmähuone") yet so you have to add a check for this to avoid errors.